components/base/base.go

```diff
@@ -59,18 +56,8 @@ type Base interface {
 	SetVelocity(ctx context.Context, linear, angular r3.Vector, extra map[string]interface{}) error
 
 	Properties(ctx context.Context, extra map[string]interface{}) (Properties, error)
-}
-
-// KinematicWrappable describes a base that can be wrapped with a kinematic model.
-type KinematicWrappable interface {
-	WrapWithKinematics(context.Context, motion.Localizer, []referenceframe.Limit) (KinematicBase, error)
-}
 
-// KinematicBase is an interface for Bases that also satisfy the ModelFramer and InputEnabled interfaces.
-type KinematicBase interface {
-	Base
-	referenceframe.ModelFramer
-	referenceframe.InputEnabled
+	Geometries(ctx context.Context) ([]spatialmath.Geometry, error)
```

Comment:
I should have made this a `resource.Shaped`

Reply:
yes!

Reply:
yes please